services/user.py

```diff
@@ -0,0 +1,43 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(username: str,
+                password: str,
+                email: str = None,
+                first_name: str = None,
+                last_name: str = None) -> None:
+    user = get_user_model().objects.create_user(username=username)
+    user.set_password(password)
+    if email:
+        user.email = email
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+
+
+def get_user(user_id: int) -> User:
+    user = User.objects.get(pk=user_id)
+    return user
+
+
+def update_user(user_id: int,
+                username: str = None,
+                password: str = None,
+                email: str = None,
+                first_name: str = None,
+                last_name: str = None) -> None:
+    user = User.objects.get(pk=user_id)
```

Comment:
It's better to use `get_user_model()` function instead of directly using `User`. This is a good practice in Django.